packages/template-retail-react-app/app/pages/checkout/index.jsx

```diff
@@ -29,11 +29,7 @@ import OrderSummary from '@salesforce/retail-react-app/app/components/order-summ
 import {useCurrentCustomer} from '@salesforce/retail-react-app/app/hooks/use-current-customer'
 import {useCurrentBasket} from '@salesforce/retail-react-app/app/hooks/use-current-basket'
 import CheckoutSkeleton from '@salesforce/retail-react-app/app/pages/checkout/partials/checkout-skeleton'
-import {
```

Comment:
FYI - just removed unused variable. NO FUNCTIONAL CHANGES in this file

Reply:
I think this usid was used before when we had to pass in the SLAS usid to the SCAPI createOrder call. We probably forgot to remove it here when we took that out.

This is safe to remove.